services/libfuzzer/libfuzzer.sh

```diff
@@ -101,6 +102,7 @@ then
   set -x
 
   FUZZER="$WORKDIR/domino-xpcshell/res/client.js"
+  HARNESS_EXT_ARGS="--transform $WORKDIR/dist/bin/transform.js"
```

Comment:
Quoting this makes it a single arg. Is that expected?

If not you probably want `HARNESS_EXT_ARGS=()` above, `HARNESS_EXT_ARGS+=("--transform" "$WORKDIR/dist/bin/transform.js")` here and `"${HARNESS_EXT_ARGS[@]}"` below.